src/components/CarouselItem.js

```diff
@@ -43,8 +43,8 @@ class CarouselItem extends PureComponent {
           minWidth: `${this.props.width}px`,
           pointerEvents: this.props.isDragging ? 'none' : null,
         }}
-        onMouseDown={this.props.isDraggingEnabled && this.onMouseDown}
-        onTouchStart={this.props.isDraggingEnabled && this.onTouchStart}
+        onMouseDown={this.props.isDraggingEnabled ? this.onMouseDown : undefined}
```

Comment:
Lint fails.
What if you use `null` instead?

Reply:
I changed to `null` and that seems to resolve the react-dom warnings. I'm still seeing a failure on `gh pages for a given PR` stage of the ci build thoug h